lib/utils/request.js

```diff
@@ -535,4 +536,31 @@ function getProxy() {
   var proxy = fhc.config.get("proxy");
   log.silly(proxy, "getProxy");
   return proxy;
+}
+
+/**
+ * This method can do a request as multipart/form-data
+ * key value pairs, files and buffers.
+ * @param url
+ * @param dataObject - { name: name, value:value }
+ * @param cb
+ * @param method
+ */
+function doRequestWithDataObjectValues(url, dataObject, cb, method) {
+
+  var fullUrl = urlUtils.resolve(getFeedHenryUrl(), url);
+
+  var opts = {
+    url : fullUrl,
+    headers : getHeaders(),
+    method: method || 'post',
+    proxy : getProxy()
+  };
+
+  var req = requestjs(opts,uploadHandlerFor(cb));
```

Comment:
The user is already informed of the body error parsed. The implementation is "uploadHandlerFor(cb)" which is the default method used in the app.